lib/smoke/verify-url.js

```diff
@@ -24,9 +25,17 @@ const verifyUrl = (testPage, browser, tests) => async () => {
 			return (typeof expectation === 'undefined' ? Promise.resolve() : Promise.resolve(testFn(testPage)).then(result => ({ name, results: [].concat(result) })));
 		});
 
-
 		const checkResults = await Promise.all(checks);
-		console.log(chalk`{bold Testing URL }({blue ${testPage.type}}): {underline.yellow ${testPage.url.toString()}}`);
+		//check for user option, use real url for logging
+		let realUrl = ''
```

Comment:
```suggestion
		let realUrl = {};
```

If the `realUrl` is subsequently constructed via `new URL()` then it looks something like:

`URL {origin: "https://ft.com", protocol: "https:", username: "", password: "", host: "ft.com", …}`

And its type is `object`.

On that basis it might make sense for this default value to be of the same type that it will become if the subsequent condition (i.e. `testPage.requestHeaders['FT-Test-Host']`) evaluates to true.